BricksAndHearts.UnitTests/ControllerTests/Landlord/LandlordControllerTests.cs

```diff
@@ -21,4 +28,19 @@ public void RegisterGet_CalledByUnregisteredUser_ReturnsRegisterViewWithEmail()
         result!.Model.Should().BeOfType<LandlordProfileModel>()
             .Which.Email.Should().Be(unregisteredUser.GoogleEmail);
     }
+
+    [Fact]
+    public async void ApproveCharter_CallsApproveLandlord()
+    {
+        // Arrange 
+        var adminUser = CreateAdminUser();
+        MakeUserPrincipalInController(adminUser, _underTest);
+        var landlord = CreateLandlordUser();
+
+        // Act
+        var result = await _underTest.ApproveCharter(landlord.Id) as ViewResult;
+
+        // Assert
+        A.CallTo(() => landlordService.ApproveLandlord(landlord.Id, adminUser)).MustHaveHappened();

```

Comment:
Do you want to test anything with the `result`?

Reply:
For example, I tried adding the line 
result!.ViewData.Model.Should().BeOfType<LandlordProfileModel?>();
but result comes back as null, so the test then fails.
I guess since result isn't currently used I could get rid of it entirely, but I think that once database-related unit tests are working it would be good to go back and add an appropriate test for the result here - leaving the unused variable here might remind whoever ends up adding missing unit tests that this unit test could do with updating.